.gitignore

```diff
@@ -20,6 +20,7 @@
 *.pem
 
 # local env files
+.env

```

Comment:
Prisma works with `.env` file, therefore it is now ignored to not leak any data. Everything else is specified in Vercel.